index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@300;400;500&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="CSS/Styles.css">
+    <title>My Personal Portfolio Project</title>
+    <link rel="icon" href="/favicon.ico" type="image/x-icon">
+</head>
+<body>
+    <header>
+        <div class="header">
+            <img class="logo" src="Images/image_geometry_1.png" alt="Top Geometric Figures">
+            <a href="#"><img class="menu" src="Images/ic_menu.png" alt="mobile-menu"></a>
+        </div>
```

Comment:
According to the evaluation guidelines, this should be a `<nav>` tag. Kindly fix this.

<img width="372" alt="Screen Shot 2022-05-25 at 17 18 56" src="https://user-images.githubusercontent.com/82426682/170377528-ec8a650f-7727-448b-9ce5-bb29e9e3006a.png">